modules/fileout_netcdf/tests/Makefile.am

```diff
@@ -74,6 +74,7 @@ $(srcdir)/package.m4: $(top_srcdir)/configure.ac
 	echo 'm4_define([AT_PACKAGE_VERSION],   [@PACKAGE_VERSION@])' && \
 	echo 'm4_define([AT_PACKAGE_STRING],    [@PACKAGE_STRING@])' && \
 	echo 'm4_define([AT_PACKAGE_BUGREPORT], [@PACKAGE_BUGREPORT@])'; \
```

Comment:
Have you tested this? Do they work? 
should you have && and remove ;
Something like: 
 echo 'm4_define([AT_PACKAGE_BUGREPORT], [@PACKAGE_BUGREPORT@])' && \

Reply:
Good catch  this did work, but I should use && there and not ;